repos/rust-lang/mdBook.toml

```diff
@@ -6,6 +6,9 @@ bots = ["rustbot"]
 [access.teams]
 devtools = "write"
 
+[access.individuals]
+Dylan-DPC = "write"
```

Comment:
I'd love for us to get in the habit of documenting why individuals have individual access so we can continually evaluate whether those conditions still exist.

Reply:
Good idea! I added a comment. I do hope this is temporary.